src/test/java/games/strategy/internal/persistence/serializable/VersionProxyAsProxyTest.java

```diff
@@ -0,0 +1,24 @@
+package games.strategy.internal.persistence.serializable;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import games.strategy.persistence.serializable.AbstractProxyTestCase;
+import games.strategy.persistence.serializable.ProxyFactory;
+import games.strategy.util.Version;
+
+public final class VersionProxyAsProxyTest extends AbstractProxyTestCase<Version> {
+  public VersionProxyAsProxyTest() {
+    super(Version.class);
+  }
+
+  @Override
+  protected Version createPrincipal() {
+    return new Version(1, 2, 3, 4);
```

Comment:
The purpose of this test is to ensure a principal can be round-tripped via its proxy.  If there's no conditional logic in how a principal's state is persisted, it should only be necessary to test a single instance.

I'm assuming your reference to covering all constructors is because, for example, using a different constructor could possibly initialize a field differently (e.g. setting it to `null`) and that may require special handling in the proxy.  I can buy that, so I'll go ahead and add support for returning multiple principal instances to be round-tripped.